### PR TITLE
Validate toml

### PR DESCRIPTION
**Pull request by jessfraz**
Maybe we should validate the toml of the Maintainers file so we don't go crazy

### CODE REVIEW

**Comment by jfrazelle** [comment]
Can you please sign your commits following these rules:

https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work

The easiest way to do this is to amend the last commit:

`git commit --amend -s --no-edit && git push -f`

**Comment by tianon** [comment]
_classic_

**Comment by jfrazelle** [comment]
hmmm it does not like my new hook :(

**Comment by jfrazelle** [comment]
ohhh i see

**Comment by tianon** [comment]
`<>` :wink:

**Comment by tianon** [comment]
LGTM

**Comment by mrunalp** [comment]
@jfrazelle Maybe have your bot ignore your github id :)

File: project/make/validate-toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+badFiles=()
+for f in "${files[@]}"; do
+	# we use "git show" here to validate that what's committed is formatted
+	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv)" ]; then
+		badFiles+=( "$f" )
+	fi
+done
+
+if [ ${#badFiles[@]} -eq 0 ]; then
+	echo 'Congratulations!  All toml source files are properly formatted.'
```
Comment (LK4D4) [inline project/make/validate-toml]: Not sure that this message is valid. Cause we checking not formatting.

Comment (tianon) [inline project/make/validate-toml]: How about "All toml source files have valid syntax." ?

Comment (LK4D4) [inline project/make/validate-toml]: SGTM

Comment (jfrazelle) [inline project/make/validate-toml]: ah nice ya

Comment (jfrazelle) [inline project/make/validate-toml]: too bad there is no tomlfmt

**Comment by tianon** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**